gulpfile.js

```diff
@@ -129,6 +130,27 @@ gulp.task('build:index', ['clean'], function buildIndex() {
   });
 });
 
+gulp.task('build:categories', ['clean'], function buildCategories() {
+  return Promise.all(parseRecipes.categories.map(function(category) {
+    var categoryRecipes = recipes.filter(function(recipe) {
+      return recipe.category === category.title;
+    }).map(function(recipe) {
+      return {
+        slug: recipe.slug,
+        title: recipe.name,
+        summary: marked(recipe.summary)
+      };
+    });
+
+    return renderFile('./src/tpl/category.html', {
```

Comment:
I am assuming that we are setting the page title? I would expect something along the lines of "Beyond Offline Recipes - ServiceWorker Cookbook" for categories and "API Analytics - Beyond Offline - Service Worker Cookbook" for recipes.